addons/mail/static/src/components/discuss_sidebar_item/discuss_sidebar_item.xml

```diff
@@ -28,6 +28,9 @@
                     <div class="o_DiscussSidebarItem_item o_DiscussSidebarItem_name" t-att-class="{ 'o-item-unread': thread.message_unread_counter > 0 }">
                         <t t-esc="thread.displayName"/>
                     </div>
+                    <t t-if="thread.mass_mailing">
+                        <i class="fa fa-envelope-o" title="Sends messages by email" role="img"/>
```

Comment:
The content of `title` is acceptable in code, but in UI the wording is off. Something like "Messages are sent by email" is a better way to formulate a UI feedback.